main/admin/course_edit.php

```diff
@@ -295,6 +295,17 @@
     true
 );
 
+if (api_get_configuration_value('allow_course_multiple_languages')) {
+    // Course Multiple language.
+    $languages = api_get_languages();
+    $cbMultiLanguage = $form->getElementByName('extra_multiple_language');
+    if (isset($cbMultiLanguage)) {
+        foreach ($languages['folder'] as $langFolder) {
+            $cbMultiLanguage->addOption($langFolder, $langFolder);
```

Comment:
We can see in the selector de language folder but we should see the translation, the folder name should be the value that is sent by the form but the user should see the translation in its language (there should be a get_lang[$langFolder] or algo asi para uno de los parametros

Reply:
Thanks @NicoDucou , the fix is add in the next commit.